src/node_native_module.cc

```diff
@@ -174,6 +174,48 @@ MaybeLocal<Function> NativeModuleLoader::CompileAsModule(
   return LookupAndCompile(context, id, &parameters, result);
 }
 
+Local<String> NativeModuleLoader::LoadInternalModuleSource(Isolate* isolate,
```

Comment:
Is a `Local` more appropriate than a `MaybeLocal`?

Reply:
I could probably move where that exception is thrown and make it a MaybeLocal

Reply:
Should this be addressed before this is considered ready?

Reply:
yes